### PR TITLE
[Buildkite] Skip install package command in serverless builds for some packages

### PR DESCRIPTION
**Pull request by mrodm**
## Proposed commit message

Skip "elastic-package install" command in Serverless builds. "elastic-package test asset" already performs the installation of the package too.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

### CODE REVIEW

File: .buildkite/scripts/common.sh [hunk 1]
```diff
@@ -814,10 +814,16 @@ run_tests_package() {
...
 
-    echo "--- [${package}] test installation"
-    if ! install_package "${package}" ; then
-        return 1
+    if ! is_serverless ; then
+        # Just run install command on non serverless projects,
+        # "elastic-paackage test asset" command already installs the package via upload API
+        # as it does "elastic-package install" command
+        echo "--- [${package}] test installation"
+        if ! install_package "${package}" ; then
```
Comment (jsoriano) [inline .buildkite/scripts/common.sh:822]: Could we keep this test for some package in serverless? I would like to keep testing this command.

Comment (mrodm) [reply .buildkite/scripts/common.sh:832]: Sure, I've added the logic to just skip one package for now here https://github.com/elastic/integrations/pull/9686/commits/01e0cb3192e4c13697ca7dbde14e8011073c5bf2

File: .buildkite/scripts/common.sh [hunk 2]
```diff
@@ -755,6 +755,19 @@ build_zip_package() {
...
+    local package=$1
+    if ! is_serverless ; then
+        return 1
+    fi
+
+    if [[ "$package" == "security_detection_engine" ]]; then
+        return 0
+    fi
+
+    return 1
```
Comment (kpollich) [inline .buildkite/scripts/common.sh:768]: I think this is fine-ish for now as this package is an outlier in terms of size and impact on memory usage. We'll have to monitor whether the duplicate installations for all other packages still impact memory enough to cause an occasional OOM on QA projects.

**Review by jsoriano — APPROVED** [review]

**Review by jsoriano — APPROVED** [review]

**Pull request merged**